bin/deploy.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 REVISION=$(git show-ref origin/master |cut -f 1 -d ' ')
 TAGGED_IMAGE=gcr.io/${GOOGLE_PROJECT}/libraries.io:${REVISION}
-gcloud --quiet container images describe ${TAGGED_IMAGE} || exit "Container not finished building"
+gcloud --quiet container images describe ${TAGGED_IMAGE} || status=$?; echo "Container not finished building" >&2; exit $status
```

Comment:
this appears to always exit even if the container is ready.

Reply:
Sorry for stuffing that up! See PR #2559 for fix.

Reply:
thank you! I probably should have just fixed it but I wasn’t sure of the right syntax and got distracted. Appreciate the original fixes and latest fix both.